terraform/eks/variables.tf

```diff
@@ -17,4 +17,7 @@ variable "eks_cluster_name" {
   default = "aoc-test-eks-ec2"
 }
 
+variable "sample_app_mode" {
```

Comment:
i think we can define this var in the common.tf so that all the platforms can leverage this flag even though they don't have the logic yet.